Metadata/Driver/AnnotationDriver.php

```diff
@@ -175,7 +180,7 @@ public function loadMetadataForClass(\ReflectionClass $class)
                         $params[] = $this->convertReferenceValue($paramName, $annot->params[$paramName]);
                     }
 
-                    if (!$params) {
+                    if (!$params && !$annot instanceof Call) {
                         continue;
```

Comment:
No, I don't remember, but yeah that seems like it could be removed. This would allow calling methods without parameters?